dbt_subprojects/solana/models/bitget_wallet/bgw_solana_swap.sql

```diff
@@ -0,0 +1,84 @@
+{{
+    config(
+
+        schema = 'bgw_solana',
+        alias = 'swap',
+        partition_by = ['block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
```

Comment:
for merge, we need to establish unique keys in the config, otherwise dbt doesn't know how to write the merge statement join conditions